### DISCUSSION TITLE
The idea to implement a browser in Comfy Desktop

### DISCUSSION THREAD

**Discussion by SunGreen777** [comfyanonymous/ComfyUI#5821] [Ideas]
![Без имени-2](https://github.com/user-attachments/assets/1ba3e3c3-54b2-42f8-aff2-e278da652410)
About the desktop version
A good example is the Fl Studio 24 browser
Browser tabs are lora c preview, nodes, models.
And also please pay attention of developers to the work of Comfy. 
The implementation is not user friendly.  
It is an interface for designer, artist, and we are constantly sent to the github.
Example. Earlier I used A1111, Forge, Focooos, invokeAi.
I have FLUX GGUG models and could not even imagine that to use them I need to download nodes.
Why make it so complicated? 
You can run a catalog scan and let the necessary nodes be downloaded automatically, or you can enable popular nodes by default. 
There are a lot of such moments.

Translated with www.DeepL.com/Translator (free version)

**Comment by ltdrdata** [comment]
1. GGUF model support is not included in the ComfyUI core because it is already actively supported through an extension. Since the manpower available to manage ComfyUI core is limited, adding features directly to the core would inevitably slow down the pace of improvements. Therefore, it is not always desirable for ComfyUI core to provide built-in support for everything.

2. Similarly, there are already excellent custom nodes available.
https://github.com/Kinglord/ComfyUI_LoRA_Sidebar
